indico/modules/events/contributions/forms.py

```diff
@@ -32,3 +32,13 @@ class ContributionForm(IndicoForm):
     duration = TimeDeltaField(_("Duration"), [DataRequired()], default=timedelta(minutes=20),
                               units=('minutes', 'hours'),
                               description=_("The duration of the contribution"))
+
+
+class ContributionProtectionForm(IndicoForm):
+    managers = PrincipalListField(_('Managers'), description=_('List of users allowed to modify the contribution'))
```

Comment:
Use `serializable=False` in this field; we don't use the default True value for it anywhere else anymore - it's a leftover from the early 1.9.x era where we stored ACLs as JSON